include/gc/Dialect/OneDNNGraph/OneDNNGraphTypes.td

```diff
@@ -39,4 +40,23 @@ def OneDNNGraph_DataType : AnyTypeOf<[
 def OneDNNGraph_LogicalTensor : TensorOf<[OneDNNGraph_DataType]>;
 def OneDNNGraph_FloatTensor : TensorOf<[OneDNNGraph_Float]>;
 
+//===----------------------------------------------------------------------===//
+// Enum types.
+//===----------------------------------------------------------------------===//
+
+def PropertyType 
```

Comment:
The name `Property` seems to be too general; a more specific name may be better.

Reply:
This just follows [onednn graph api's name](https://oneapi-src.github.io/oneDNN/enum_dnnl_graph_logical_tensor_property_type.html#doxid-classdnnl-1-1graph-1-1logical-tensor-1a037ba7c242d8127d2f42c0c2aef29d82), maybe we can name it as TensorPropertyType?